### ISSUE TITLE
Translation legend - integrate in language files (qm files)

### ISSUE THREAD

**Issue by sjib** [QGEP/QGEP#551]
In meinem Falle funktioniert die Übersetzung soweit gut mit Ausnahme der Legenden, die unverändert in Englich sind. Wie kann ich dies anpassen?
![legende](https://user-images.githubusercontent.com/1265865/75558790-a624ea80-5a42-11ea-9f0f-8be96b677d4e.png)

**Comment by m-kuhn** [comment]
We will need to integrate (rule based) renderer titles into translation.

- @signedav can you schedule that for QGIS 3.14?
- Meanwhile, we'll need to update the projects by hand

**Comment by TriformMoritz** [comment]
Yes definitely! I need a french translation. AND also for the other objects (reaches, catchment).
Would be nice to have a few more styles too (for example: ws_owner, necessite_assainir).

**Comment by sjib** [comment]
> We will need to integrate (rule based) renderer titles into translation. @signedav can you schedule that for QGIS 3.14?
@signedav @m-kuhn Is this already implemented in the meantime?

If yes, how can we do the translation on transifex?

**Comment by m-kuhn** [comment]
No, no changes so far

**Comment by sjib** [comment]
@m-kuhn Is there an QGIS issue for this? Or has this been implemented already? Else can you create one?

**Comment by m-kuhn** [comment]
Added to https://github.com/qgis/QGIS/issues/36729
